src/app/pools/components/PoolForm/PoolForm.test.tsx

```diff
@@ -34,25 +35,22 @@ describe("PoolForm", () => {
   });
 
   it("can render", () => {
-    const store = mockStore(state);
-
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter initialEntries={["/"]}>
-          <CompatRouter>
-            <PoolForm />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    renderWithMockStore(
```

Comment:
You could use `renderWithBrowserRouter` to save having to set up the router here. Something like this:
```typescript
renderWithBrowserRouter(<PoolForm />, {
    route: "/",
    wrapperProps: { state },
  });
```

Reply:
Oh nice!!! Implementing that now